### ISSUE TITLE
problem

### ISSUE THREAD

**Issue by CHSLAM** [Zhengjun-Du/LC-CRF-SLAM#4]
I   have a question if LC-CRF can work  without GCRANSAC?

**Comment by Zhengjun-Du** [comment]
> 

Yes, of cause，typically this model is slower than CRF model,  you can remove it if the accuracy loss is accepted.
